### ISSUE TITLE
StoredCdr redesign into CDR

### ISSUE THREAD

**Issue by danbogos** [cgrates/cgrates#312]
Since StoredCdr is one of the oldest structures in CGRateS today came the need to be refactored.

First of all the name will be changed into CDR to reflect internal CDR structure.

Field name changes:
 * CgrId -> CGRID to cope more with CamelCase rules as well as be golint compatible.
 * OrderId -> OrderID - to cope with golint.
 * MediationRunId -> RunID - mediation concept was swallowed by rating, so we remove it's references slowly.
 * CdrHost -> OriginHost - rename it to be more generic (eg: used in other events also).
 * CdrSource -> Source - rename it to be more generic and use it in other events also.
 * AccId -> OriginID: should refelct ID on the originator side and the new name suits better and also golint compatible.
 * Pdd -> PDD - golint compatible /CamelCase.
 *

**Comment by danbogos** [comment]
Have removed RatedAccount and RatedSubject since they were valid to differentiate rated from non rated CDR information. With the unified cdrs table structure this is differentiated per CDR record based on RunID.

Part of this change I have also renamed CDRE.ExportDir configuration into CDRE.ExportFolder since dir can be mixed with direction.

DerivedChargers received also small modifications for RunId -> RunID; DestinationIds ->DestinationIDs, Pdd->PDD.

Due to CDR modifications I have also updated the CDR filters sent in GetCdr and ExportCdr APIs (v2 only).

DanB
